backend/typescript/swagger.yml

```diff
@@ -438,6 +438,24 @@ paths:
         '500':
           description: Invalid request
   
+  /auth/{uid}:
+      get:

```

Comment:
shouldn't we guard this under `bearerAuth` as well?

Reply:
Same as above ^^. We need this route before the user has logged in. We need this route to get the firebaseUser to check if the user is already verified and get their email. A more secure way could be to remove this route and ask the user to enter their email. But we still need to check if the user is verified or not without them being logged in.